### PR TITLE
fix timetable travis + conflict v1.0

### PR DESCRIPTION
**Pull request by Poroma**
fix timetable Travis + conflict

### CODE REVIEW

**Comment by AndiAndreas** [comment]
Travis not fixed yet.

File: test/model/course_test.dart [hunk 1]
```diff
@@ -191,6 +213,39 @@ void main(){
     });
   });
 
+  test('12', () {
```
Comment (RedTo) [inline test/model/course_test.dart]: There is a ')' missing at every test...

Comment (RedTo) [reply test/model/course_test.dart]: That's the reason why travis is failing. So just close all the opend brackets and travis should be fixed.

**Comment by AndiAndreas** [comment]
This is not how we like to have it. We need to do it like in Figma. Or in a other beautiful way. Although in this approach it is not possible to have multiple lecture times per week.

Next steps will be: Merge current master into branch. Refactor code that we can have multiple lectures per week. Improve GUI.

**Comment by RedTo** [comment]
Please also resolve the merge conflicts.
I think you just have to delete pubspec.lock, but I'm not quite sure.

**Comment by RedTo** [comment]
Sorry for confusing you.
I just wanted to give you a little help.

First of all you should merge the master into your feature branch. Before or after merging you should resolve the conflicts. I think this could be done by deleting the pubspec.lock, because on master there is no pubspec.lock anymore.
After that, you can change the UI that it fits the [figma design prototype](https://www.figma.com/proto/LIvZQnfyBeJ4f9J6j4LV4Z/CiE-App?node-id=71%3A39&scaling=contain&redirected=1). And like Andi mentioned refactor the code, that we can have mulitple lectures per week.

Hope now everything is clear... Sorry again, if I have confused you. Please just comment below if anything is unclear.

Happy coding!

**Comment by Poroma** [comment]
alright I did what you told thank you

File: lib/model/course/course.dart [hunk 2]
```diff
@@ -1,27 +1,310 @@
...
     "13", "14"
   ];
-}
\ No newline at end of file
+}
+
+enum CourseAvailability { AVAILABLE, PENDING, UNAVAILABLE }
+enum Weekday { Mon, Tue, Wed, The, Fri, Sat, Sun }
+enum Campus { KARLSTRASSE, LOTHSTRASSE, PASING }
+//enum Faculty {
```
Comment (RedTo) [inline lib/model/course/course.dart]: Just for documentation, those comments will stay. We will rebuild this class later based on those things.

File: lib/model/course/course.dart [hunk 3]
```diff
@@ -1,27 +1,310 @@
...
   static const List<String> DEPARTMENTS = [
     "01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12",
     "13", "14"
   ];
-}
\ No newline at end of file
+}
+
+enum CourseAvailability { AVAILABLE, PENDING, UNAVAILABLE }
+enum Weekday { Mon, Tue, Wed, The, Fri, Sat, Sun }
```
Comment (RedTo) [inline lib/model/course/course.dart]: Please change The to Thu

File: lib/model/course/course.dart [hunk 4]
```diff
@@ -1,27 +1,310 @@
...
+      case 1:
+        return CourseAvailability.PENDING;
+      case 2:
+        return CourseAvailability.UNAVAILABLE;
+    }
+    return CourseAvailability.AVAILABLE;
+  }
+}
+
+
```
Comment (RedTo) [inline lib/model/course/course.dart]: Empty lines, could be removed. Just if you like :-)

File: lib/model/course/course.dart [hunk 5]
```diff
@@ -1,27 +1,310 @@
...
+class WeekdayUtility {
+  static String getWeekdayAsString(Weekday day) {
+    switch (day) {
+      case Weekday.Mon:
+        return "Mon";
+      case Weekday.Tue:
+        return "Tue";
+      case Weekday.Wed:
+        return "Wed";
+      case Weekday.The:
```
Comment (RedTo) [inline lib/model/course/course.dart]: Weekday.Thu

File: lib/model/course/course.dart [hunk 6]
```diff
@@ -1,27 +1,310 @@
...
+
+  static int getWeekdayAsInt(Weekday day) {
+    switch (day) {
+      case Weekday.Mon:
+        return 0;
+      case Weekday.Tue:
+        return 1;
+      case Weekday.Wed:
+        return 2;
+      case Weekday.The:
```
Comment (RedTo) [inline lib/model/course/course.dart]: Weekday.Thu

File: lib/model/course/course.dart [hunk 7]
```diff
@@ -1,27 +1,310 @@
...
+  static Weekday intToWeekday(int day) {
+    switch (day) {
+      case 0:
+        return Weekday.Mon;
+      case 1:
+        return Weekday.Tue;
+      case 2:
+        return Weekday.Wed;
+      case 3:
+        return Weekday.The;
```
Comment (RedTo) [inline lib/model/course/course.dart]: Weekday.Thu

**Comment by AndiAndreas** [comment]
https://travis-ci.org/mobileappdevhm/dev-team-1-cie-app-in-flutter/builds/389405281
Build Fnished. Travis build for pull request stucks. But the build above is the same one.

**Pull request merged**